tools/flake8-torch/flake8_torch/checker.py

```diff
@@ -1,19 +1,52 @@
 from pathlib import Path
 import yaml
 import libcst as cst
+import libcst.codemod as codemod
+from dataclasses import dataclass
+from typing import Optional, List
+import argparse
+import contextlib
+import io
+import sys
 
 __version__ = "0.0.1"
 
+CYAN = "\033[96m"
+RED = "\033[31m"
+BOLD = "\033[1m"
+ENDC = "\033[0m"
```

Comment:
Don't emit tty escape codes to pipes or on Windows
```suggestion
IS_TTY = hasattr(sys.stdout, "isatty") and sys.stdout.isatty()
CYAN = "\033[96m" if IS_TTY else ""
RED = "\033[31m" if IS_TTY else ""
BOLD = "\033[1m" if IS_TTY else ""
ENDC = "\033[0m" if IS_TTY else ""
```